bundle/get_started.md

```diff
@@ -0,0 +1,417 @@
+
+# Get started to MONAI bundle
+
+A MONAI bundle usually includes the stored weights of a model, TorchScript model, JSON files which include configs and metadata about the model, information for constructing training, inference, and post-processing transform sequences, plain-text description, legal information, and other data the model creator wishes to include.
+
+For more information about MONAI bundle, please read the description: https://docs.monai.io/en/latest/bundle_intro.html.
+
+This is a step-by-step tutorial to help get started to develop a bundle package, which contains a config file to construct the training pipeline and also has a `metadata.json` file to define the metadata information.
+
+Mainly contains the below sections:
+- Define a training config with `JSON` or `YAML` format.
+- Execute training based on bundle scripts and configs.
+- Execute other scripts for bundle functionalities.
+- Hybrid programming with config and python code.
+
+You can find the usage examples of MONAI bundle key features and syntax in this tutorial, like:
+- Instantiate a python object from a dictionary config with `_target_` indicating class or function name or module path.
+- Execute python expression from a string config with the `$` syntax.
+- Refer to other python object with the `@` syntax.
+- Macro text replacement with the `%` syntax to simplify the config content.
+- Leverage the `_disabled_` syntax to tune or debug different components.
+- Override config content at runtime.
+- Hybrid programming with config and python code.
+
+## Download dataset
+
+Downloads and extracts the dataset for this example.
+The dataset comes from http://medicaldecathlon.com/.
+Here specify a directory with the `MONAI_DATA_DIRECTORY` environment variable to save downloaded dataset and outputs, if no environment, save to the temorary directory.
+
+```python
+import os
+import tempfile
+from monai.apps import download_and_extract
+
+resource = "https://msd-for-monai.s3-us-west-2.amazonaws.com/Task09_Spleen.tar"
+md5 = "410d4a301da4e5b2f6f86ec3ddba524e"
+
+directory = os.environ.get("MONAI_DATA_DIRECTORY")
+root_dir = tempfile.mkdtemp() if directory is None else directory
+compressed_file = os.path.join(root_dir, "Task09_Spleen.tar")
+data_dir = os.path.join(root_dir, "Task09_Spleen")
+if not os.path.exists(data_dir):
+    download_and_extract(resource, compressed_file, root_dir, md5)
+```
+
+## Define train config - Set imports and input / output environments
+
+Now let's start to define the config file for a regular training task. MONAI bundle support both `JSON` and `YAML` format, here we use `JSON` as the example. the whole config for training is available and can be a reference: https://github.com/Project-MONAI/tutorials/blob/main/bundle/spleen_segmentation/configs/train.json.
```

Comment:
Can you either make the hyperlink in a new line, or make it a relative link?

It will then be easier for me to either replace the line elsewhere, or copy the entire folder without Github reference.

Thanks!

Reply:
updated in the latest commit.

Thanks.